originals/a.yaml

```
@@ -80,8 +80,15 @@
     - Turn-Based Strategy
 
 - name: The Adventures of Robbo
+  names:
+  - Robbo
   external:
     wikipedia: The Adventures of Robbo
```

```suggestion
    wikipedia: Robbo (video game)
```
also let's move this to `r.yaml`